### DISCUSSION TITLE
wire:ignore is not working with wire:navigate

### DISCUSSION THREAD

**Discussion by nunowar** [livewire/livewire#8155] [1. Help] [answered]
**Livewire version**
v3.4.4

**Laravel version**
v10.3.3

**Which PHP version are you using?**
PHP 8.3

I need help troubleshooting an issue with a carousel menu. It features multiple categories, but clicks cause the menu to reload, despite using wire:ignore. What I might be doing wrong?

**Steps To Reproduce**

1. Click in the next button
2. Click in one category, and you will see the menu reloading

**Repository or Wirebox URL**

https://wirebox.app/b/x9oln

**Comment by joshhanley** [comment, marked as answer]
@nunowar `wire:ignore` is only for within a page. Navigate is a page change so nothing will be preserved unless you use `@persist`.

**Reply by nunowar** [reply]
Thanks. With @persist works well. 

I updated the wirebox: https://wirebox.app/b/x9oln
